hoot-core-test/src/test/cpp/hoot/core/io/ServicesDbTestUtils.cpp

```diff
@@ -89,9 +89,9 @@ void ServicesDbTestUtils::deleteDataFromOsmApiTestDatabase()
   database.close();
 }
 
-QUrl ServicesDbTestUtils::getDbModifyUrl()
+QUrl ServicesDbTestUtils::getDbModifyUrl(const QString& mapName)
 {
-  return QUrl(HootApiDb::getBaseUrl().toString() + "/testMap");
+  return QUrl(HootApiDb::getBaseUrl().toString() + "/" + mapName);
```

Comment:
Use the supplied map name instead of `testMap` so that each test can create its own unique map and not clobber each other.  The default parameter value is `testMap` so in some cases where it doesn't matter it acts as before.